docs/detections/rules-coverage.asciidoc

```diff
@@ -8,7 +8,7 @@
 
 The **MITRE ATT&CK® coverage** page shows which https://attack.mitre.org[MITRE ATT&CK®] adversary tactics and techniques are covered by your installed and enabled detection rules.
 
-Mirroring the MITRE ATT&CK® framework, columns represent a major tactics, and cells within each column represent a tactic's related techniques. Cells are colored progressively darker when more rules are enabled for each technique, as indicated in the **Legend** at the top.
+Mirroring the MITRE ATT&CK® framework, columns represent major tactics, and cells within each column represent a tactic's related techniques. Cells are darker when more rules are enabled for each technique, as indicated in the **Legend** at the top.
 
 NOTE: This page only includes the detection rules you currently have installed. If there are Elastic prebuilt rules you haven't installed yet, they will not be included in determining your MITRE ATT&CK® coverage.
```

Comment:
We also do not show rules that do not have MITRE mappings.

Reply:
```suggestion
[NOTE]
====
This page only includes the detection rules you currently have installed, and only rules that are mapped to MITRE ATT&CK® tactics. If there are Elastic prebuilt rules you haven't installed yet, or custom rules you haven't mapped to a tactic, they will not be included in determining your MITRE ATT&CK® coverage.

You can map custom rules to tactics in **Advanced settings** when creating or editing a rule.
====
```